vulcanai/models/basenetwork.py

```diff
@@ -494,6 +538,59 @@ def _assert_same_devices(self, comparison_device=None):
                     comparison_device,
                     incompatible_collector))
 
+    # modified from
+    # https://github.com/Bjarten/early-stopping-pytorch/blob/master
+    # /pytorchtools.py
+    class EarlyStopping:
+        """Early stops the training if validation loss doesn't improve after
+        a given patience."""
+
+        def __init__(self, patience=2, verbose=False):
+            """
+            Args:
+                patience (int): How long to wait after last time validation
```

Comment:
by "how long" imi assuming you are referring to the # of epochs passed? Let's clarify that iim the docs please